identifiers/country-gb/constituencies.csv

```diff
@@ -1,651 +1,863 @@
-ocd-division/country:gb/part:sct/ed:west_dunbartonshire,West Dunbartonshire
-ocd-division/country:gb/part:wls/ed:aberavon,Aberavon
-ocd-division/country:gb/part:wls/ed:aberconwy,Aberconwy
-ocd-division/country:gb/part:wls/ed:alyn_and_deeside,Alyn and Deeside
-ocd-division/country:gb/part:wls/ed:arfon,Arfon
-ocd-division/country:gb/part:wls/ed:blaenau_gwent,Blaenau Gwent
-ocd-division/country:gb/part:wls/ed:brecon_and_radnorshire,Brecon and Radnorshire
-ocd-division/country:gb/part:wls/ed:bridgend,Bridgend
-ocd-division/country:gb/part:wls/ed:caerphilly,Caerphilly
-ocd-division/country:gb/part:wls/ed:cardiff_central,Cardiff Central
-ocd-division/country:gb/part:wls/ed:cardiff_north,Cardiff North
-ocd-division/country:gb/part:wls/ed:cardiff_south_and_penarth,Cardiff South and Penarth
-ocd-division/country:gb/part:wls/ed:cardiff_west,Cardiff West
-ocd-division/country:gb/part:wls/ed:carmarthen_east_and_dinefwr,Carmarthen East and Dinefwr
-ocd-division/country:gb/part:wls/ed:carmarthen_west_and_south_pembrokeshire,Carmarthen West and South Pembrokeshire
-ocd-division/country:gb/part:wls/ed:ceredigion,Ceredigion
-ocd-division/country:gb/part:wls/ed:clwyd_south,Clwyd South
-ocd-division/country:gb/part:wls/ed:clwyd_west,Clwyd West
-ocd-division/country:gb/part:wls/ed:cynon_valley,Cynon Valley
-ocd-division/country:gb/part:wls/ed:delyn,Delyn
-ocd-division/country:gb/part:wls/ed:dwyfor_meirionnydd,Dwyfor Meirionnydd
-ocd-division/country:gb/part:wls/ed:gower,Gower
-ocd-division/country:gb/part:wls/ed:islwyn,Islwyn
-ocd-division/country:gb/part:wls/ed:llanelli,Llanelli
-ocd-division/country:gb/part:wls/ed:merthyr_tydfil_and_rhymney,Merthyr Tydfil and Rhymney
-ocd-division/country:gb/part:wls/ed:monmouth,Monmouth
-ocd-division/country:gb/part:wls/ed:montgomeryshire,Montgomeryshire
-ocd-division/country:gb/part:wls/ed:neath,Neath
-ocd-division/country:gb/part:wls/ed:newport_east,Newport East
-ocd-division/country:gb/part:wls/ed:newport_west,Newport West
-ocd-division/country:gb/part:wls/ed:ogmore,Ogmore
-ocd-division/country:gb/part:wls/ed:pontypridd,Pontypridd
-ocd-division/country:gb/part:wls/ed:preseli_pembrokeshire,Preseli Pembrokeshire
-ocd-division/country:gb/part:wls/ed:rhondda,Rhondda
-ocd-division/country:gb/part:wls/ed:swansea_east,Swansea East
-ocd-division/country:gb/part:wls/ed:swansea_west,Swansea West
-ocd-division/country:gb/part:wls/ed:torfaen,Torfaen
-ocd-division/country:gb/part:wls/ed:vale_of_clwyd,Vale of Clwyd
-ocd-division/country:gb/part:wls/ed:vale_of_glamorgan,Vale of Glamorgan
-ocd-division/country:gb/part:wls/ed:wrexham,Wrexham
-ocd-division/country:gb/part:wls/ed:ynys_môn,Ynys Môn
+id,name,sameAs,sameAsNote,validThrough,validFrom
+ocd-division/country:gb/part:eng/region:ukc/ed:berwick-upon-tweed,Berwick-upon-Tweed,ocd-division/country:gb/part:eng/region:ukc/ed:north_northumberland,"2024-07-04: The boundaries of Berwick-upon-Tweed will change, and it will be renamed North Northumberland.",,
```

Comment:
Lets add a `validFrom` date set to `2024-07-04` as this will go into affect after the general election.

```suggestion
ocd-division/country:gb/part:eng/region:ukc/ed:berwick-upon-tweed,Berwick-upon-Tweed,ocd-division/country:gb/part:eng/region:ukc/ed:north_northumberland,"2024-07-04: The boundaries of Berwick-upon-Tweed will change, and it will be renamed North Northumberland.",,2024-07-04
```

Same for the canonical id `ocd-division/country:gb/part:eng/region:ukc/ed:north_northumberland`.

Reply:
@HKSenior you mean for all sameAs entries?

Reply:
Only for `ocd-division/country:gb/part:eng/region:ukc/ed:north_northumberland` and its non-canonical ocd id.

Reply:
@HKSenior 
Just "2024-07-04" as validFrom for `ocd-division/country:gb/part:eng/region:ukc/ed:north_northumberland` and `ocd-division/country:gb/part:eng/region:ukc/ed:berwick-upon-tweed`?

I don't know exactly what do you mean with " non-canonical "

Reply:
Sorry for the confusion here but yes.

Because the electoral district of North Northumberland won't officially exist until the next general election lets add a `validFrom` date of `2024-07-04` for the 2 relevant OCD IDs. Essentially the following changes:


```
ocd-division/country:gb/part:eng/region:ukc/ed:berwick-upon-tweed,Berwick-upon-Tweed,ocd-division/country:gb/part:eng/region:ukc/ed:north_northumberland,"2024-07-04: The boundaries of Berwick-upon-Tweed will change, and it will be renamed North Northumberland.",,2024-07-04
```

```
ocd-division/country:gb/part:eng/region:ukc/ed:north_northumberland,North Northumberland,,,,2024-07-04
```

Reply:
done, please have a look